env.js

```diff
@@ -71,12 +71,60 @@ module.exports = (function () {
   };
 
   /**
-   * A JSON object that describes where to store intermediate files (data files to be processed)
-   * Known types are 'local' and 'sandcastle'.
+   * Credentials for AWS.  Only required if env.storage.default is 'aws/s3'.
    *
-   * 'local' is the default and just stores files locally
+   * The AWS_CREDENTIALS value should be a JSON string hash containing the
+   * properties 'accessKeyId' and 'secretAccessKey'.
+   *
+   * An example of a complete AWS_CREDENTIALS value:
+   *
+   * '{"accessKeyId": "<your-access-key-id>", "secretAccessKey": "<your-secret-access-key>"}'
+   */
+  env.awsCredentials = JSON.parse(config.fromEnvironment('AWS_CREDENTIALS', '{}'));
```

Comment:
Agree. 

I guess we just also need to consider the local/dev experience too and how the env variables are surfaced in the runservers script. We can consider this as we clean this stuff up for the open beta though -  I just thought of it now when seeing the new AWS_CREDENTIALS var